sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -460,6 +460,65 @@ public partial class CustomAnalyzer : Azure.Search.Documents.Indexes.Models.Lexi
         public System.Collections.Generic.IList<Azure.Search.Documents.Indexes.Models.TokenFilterName> TokenFilters { get { throw null; } }
         public Azure.Search.Documents.Indexes.Models.LexicalTokenizerName TokenizerName { get { throw null; } set { } }
     }
+    public partial class CustomEntity
```

Comment:
These were added in Azure/azure-rest-api-specs#11113 and, it seems, never picked up (we hadn't generated isnce Oct. 2020).